runtime/tokio_util.rs

```diff
@@ -1,5 +1,46 @@
 // Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
 
+use core::future::Future;
+use core::marker::PhantomData;
+use tokio::runtime::Runtime;
+use tokio::task::JoinHandle;
+
+pub struct LocalRuntime {
+  inner: Runtime,
+  _not_send: PhantomData<*mut ()>,
+}
+
+impl LocalRuntime {
+  pub fn new() -> Self {
+    Self {
+      inner: create_basic_runtime(),
+      _not_send: PhantomData,
+    }
+  }
+
+  pub fn spawn<F>(&self, fut: F) -> JoinHandle<F::Output>
+  where
+    F: 'static + Future + Send,
+    // We could eliminate this one if necessary, but it's tedious to do so.
```

Comment:
This was suggested by @darksonn. We should move it to `deno_core` and document it as a required way of using `JsRuntime` with Tokio. It is required to finish https://github.com/denoland/deno/pull/19017